plugin/panels.py

```diff
@@ -0,0 +1,20 @@
+from .core.windows import ensure_server_panel
+from .diagnostics import ensure_diagnostics_panel
+from sublime import error_message
+from sublime_lib import Panel
+from sublime_plugin import WindowCommand
+
+assert ensure_server_panel
+assert ensure_diagnostics_panel
+
+
+class LspTogglePanel(WindowCommand):
+    def run(self, panel_type: str) -> None:
+        if not isinstance(panel_type, str):
```

Comment:
Even though theoretically users can put anything in the command arguments, I don't really feel the need for us to protect them from invalid input (especially as it's a silent failure now). Otherwise, the argument should be typed as `Any` ?